airbyte-config/models/src/main/java/io/airbyte/config/EnvConfigs.java

```diff
@@ -232,6 +238,38 @@ public WorkspaceRetentionConfig getWorkspaceRetentionConfig() {
     return new WorkspaceRetentionConfig(minDays, maxDays, maxSizeMb);
   }
 
+  @Override
+  public List<WorkerPodToleration> getWorkerPodTolerations() {
+    // Format:
+    // Tolerations separated by ;;
```

Comment:
What's the reason for using `;;` as the delimiter?

Reply:
No specific reason, I usually use it in similar parsing. I can change to just one `;` if you prefer

Reply:
I changed to just ; btw